addons/copy-reporter/userscript.js

```diff
@@ -0,0 +1,77 @@
+export default async function ({ addon, console, msg }) {
+  addon.tab.createEditorContextMenu(
+    (ctx) => {
+      // ugly hack to get the string value
+      let element = ctx.target.lastChild.lastChild;
+
+      if (ctx.type === "monitor_slider") {
+        element = element.children[element.children.length - 2].lastChild;
+      } else {
+        element = element.lastChild;
+        if (ctx.type === "monitor_default") {
+          element = element.lastChild;
+        }
+      }
+
+      if (element.innerText.length !== 0) {
+        navigator.clipboard.writeText(element.innerText);
+      }
+    },
+    {
+      className: "copy",
+      types: ["monitor_default", "monitor_large", "monitor_slider"],
+      position: "monitor",
+      order: 0,
+      label: msg("copy-value"),
+    }
+  );
+
+  // add button to reporter bubble
+  const ScratchBlocks = await addon.tab.traps.getBlockly();
+
+  // https://github.com/LLK/scratch-blocks/blob/893c7e7ad5bfb416eaed75d9a1c93bdce84e36ab/core/workspace_svg.js#L979
+  ScratchBlocks.WorkspaceSvg.prototype.reportValue = function (id, value) {
+    let block = this.getBlockById(id);
+    if (!block) {
+      throw "Tried to report value on block that does not exist.";
+    }
+
+    ScratchBlocks.DropDownDiv.hideWithoutAnimation();
+    ScratchBlocks.DropDownDiv.clearContent();
+
+    let contentDiv = ScratchBlocks.DropDownDiv.getContentDiv();
+
+    let valueReportBox = document.createElement("div");
+    valueReportBox.setAttribute("class", "valueReportBox");
+    valueReportBox.innerText = value;
+    // use to get focus and event priority
+    valueReportBox.setAttribute("tabindex", "0");
+    // if the user pressed Ctrl+C, prevent propagation to Blockly
+    valueReportBox.onkeydown = (event) => {
+      if ((event.altKey || event.ctrlKey || event.metaKey) && event.code === "KeyC") {
+        event.stopPropagation();
+      }
+    };
+
+    if (value.length !== 0) {
```

Comment:
```suggestion
    if (value.length !== 0 && !addon.self.disabled) {
```
You can add this here and `"dynamicDisable": true` to `addon.json` to allow the addon to be disabled without reloading the page.

Reply:
I can't find a way to remove the context menu entry.

Reply:
I think that's done automatically by the API.

Reply:
The other context menu plugin didn't seem to but maybe dynamicDisable isn't set right.

Reply:
"Move costume to top or bottom" supports dynamic disable and it doesn't have any extra code for it.